blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -371,33 +371,6 @@ namespace blueprint
          **/
         BundleEvalCallback afterBundleEval;
 
-        //==============================================================================
```

Comment:
Removed this stuff here as easier to type EventBridge without this stuff (less ts-ignore statements etc.). We can close this once this PR is merged: https://github.com/nick-thompson/blueprint/issues/97

Reply:
Oh sweet, thank you!